### ISSUE TITLE
Context ctor that accepts ConfigurationOptions

### ISSUE THREAD

**Issue by alberto-riggi** [thepirat000/CachingFramework.Redis#27]
Is it possible to create a constructor overload that accepts a configuration options object?
In that case it's possible to create/manipulate a config object and avoid having to serialize it before passing it to context.
`var config = new ConfigurationOptions();`
```
//or even this if you want to programmatically modify properties of the stored configuration
//var config = ConfigurationOptions.Parse(cacheConfiguration);
```
```
config.AbortOnConnectFail = false;
var context = new Context(config);
```
```
//this is the trick that I have to do currently
//var context = new Context(config.ToString());
```
I know the trick it's not the end of the world, but it'd be nicer to avoid it.

Great job by the way.
Thanks and regards

**Comment by thepirat000** [comment]
Thanks for your comments!

A new constructor overload for `Context` is exposed so you can pass the `ConfigurationOptions` object.

This was implemented in version 7.1.4, please update your references to use the latest version.

Regards

**Comment by alberto-riggi** [comment]
wow, that's agile man.
Thanks
Regads
